target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,7 @@ ramips_setup_interfaces()
 	gehua,ghl-r-001|\
 	hiwifi,hc5962|\
 	xiaomi,mir3p|\
+	xiaomi,mi-router-ac2100|\
```

Comment:
It might seem weird but `sort` says "mi-router-ac2100" should go before "mir3p".
```
$ echo -e "xiaomi,mir3p\nxiaomi,mi-router-ac2100" | sort
xiaomi,mi-router-ac2100
xiaomi,mir3p
```

Reply:
Yes. I was asked in previous comment. I think there should be exception for this device. In computers dash comes before letter. Thats why it's coming before mir3g. I'm not sure to take it before mir3g. We should decide something in here.